src/civetweb.c

```diff
@@ -6235,6 +6243,9 @@ static void addenv(struct cgi_environment *env, const char *fmt, ...)
 	char *added;
 	va_list ap;
 
+	if (env == NULL) {
```

Comment:
The function already defines a call with `env == NULL` is not valid:

    /* Append VARIABLE=VALUE\0 string to the buffer, and add a respective
     * pointer into the vars array. Assumes env != NULL and fmt != NULL. */
    static void addenv(struct cgi_environment *env, const char *fmt, ...)